WhitehillFamily.py

```diff
@@ -26,3 +26,8 @@ def spawn_child(parents):
     second_child.start()
     second_child.join()
     print('WELCOME {}'.format(second_child.name)) # Output: Hazel Witehill - 2022
+
+    third_child = ctx.Process(target=spawn_child, args=(NickAndMelissa,))
+    third_child.start()
+    third_child.join() # Expected to join in May 31 2024
+    print('WELCOME {}'.format(third_child.name))
```

Comment:
Nit -- use f-strings

Reply:
How do you think the process was started?